src/ViewLayer/RaceModeDisplay/RaceModeFaults/FaultList.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <QVector>
+#include "../FaultLabel/FaultLabel.h"
+#include "../DataLayer/MotorFaultsData/ErrorFlags.h"
+#include "../DataLayer/MotorFaultsData/LimitFlags.h"
+
+class FaultList
+{
+public:
+    FaultList();
+
+    ~FaultList();
+
+    FaultLabel getHighestActivePriorityLabel() const;
+    QVector<FaultLabel> getErrorLabelList() const;
+    QVector<FaultLabel> getLimitLabelList() const;
+    void updateErrors(const ErrorFlags& errorFlags);
+    void updateLimits(const LimitFlags& limitFlags);
+
+private:
+
+    QVector<FaultLabel> errorList_;
```

Comment:
Since this is a vector, it's a little bit misleading to call this a `errorList`.